pyleecan/Methods/Converter/Convert/Step/select_hole_rules.py

```diff
@@ -49,14 +49,40 @@ def select_hole_rules(self, is_stator):
 
         elif isinstance(hole, HoleM61):
```

Comment:
Check that all magnets are handled for all Holes
Sort elif by type number for readibility (or find a better/clever way => dict{class}=nb_magnet + call method add_rule_X to remove elif